src/pages/[quizId]/play.tsx

```diff
@@ -23,6 +26,15 @@ const Play = () => {
     teamData: state.teamData,
   }));
 
+  // for update of the quiz status through websocket
+  useSubscribeToQuizTeamUpdateStore();
```

Comment:
Wouldnt it be nicer to have that subscription directly in the `useTeamQuizDataStore` hook?

Reply:
I thought about it. I'll need to check usage again